tests/integ/modin/resample/test_resample.py

```diff
@@ -357,3 +357,17 @@ def test_resample_date_trunc_hour():
         lambda df: df.resample("2H").min(),
         check_freq=False,
     )
+
+
+# One extra query to convert index to native pandas for dataframe constructor
+@pytest.mark.parametrize("q", [0.1, 0.7])
+@sql_count_checker(query_count=3, join_count=1)
+def test_resample_quantile_various_q(q):
+    eval_snowpark_pandas_result(
+        *create_test_dfs(
+            {"A": np.random.randn(15)},
+            index=native_pd.date_range("2020-01-01", periods=15, freq="1s"),
+        ),
+        lambda df: df.resample(rule="1s").quantile(q=q),
```

Comment:
Should we resample to a frequency that's different from the input index's frequency?

Reply:
See the comment above we have tests for that

Reply:
True, but when each sample group has just one row, all the quantiles here yield the same result. Can we resample to `3s` instead?

Reply:
Ah I see your point, my bad. Yep, fixed.